gradle/versions.gradle

```diff
@@ -46,14 +38,19 @@ dependencyManagement {
     dependency 'org.apache.logging.log4j:log4j-core:2.13.0'
     dependency 'org.apache.logging.log4j:log4j-slf4j-impl:2.13.0'
 
-    dependency 'org.apache.tuweni:tuweni-net:0.10.0'
-    dependency 'org.apache.tuweni:tuweni-toml:0.10.0'
+    dependencySet(group: 'org.apache.tuweni', version: '0.10.0') {
+      entry 'tuweni-net'
```

Comment:
nit: why specify the versions this way? is this better or needed?

Reply:
its how teku does it - bad answer.